app/src/main/java/org/jellyfin/androidtv/ui/TextButton.kt

```diff
@@ -0,0 +1,39 @@
+package org.jellyfin.androidtv.ui
+
+import android.content.Context
+import android.view.Gravity
+import android.view.View.OnFocusChangeListener
+import android.view.ViewGroup
+import androidx.appcompat.widget.AppCompatButton
+import androidx.core.content.res.ResourcesCompat
+import org.jellyfin.androidtv.R
+import org.jellyfin.androidtv.util.Utils
+
+/**
+ * Button used in [JumpList].
+ */
+class TextButton(
+	context: Context,
+	text: String?,
+	size: Float,
+	listener: OnClickListener?
+) : AppCompatButton(context) {
+	init {
+		// Set box size
+		layoutParams = Utils.convertDpToPixel(context, size).let { pixels ->
+			ViewGroup.LayoutParams(pixels * 2 + 15, pixels + 40)
+		}
+
+		setBackgroundColor(0)
+		textSize = size
```

Comment:
It looks like the old code had a check that would set `size` to 16 if it was 0. Maybe we should continue to default that?

Reply:
I looked at the call sides and it never passes `0` so I removed that check.